commands/proto.go

```diff
@@ -93,13 +97,120 @@ extend google.protobuf.MessageOptions {
 	}
 
 	return output
+}
+
+func (cmd *GenerateProtoCmd) emitOptionsHeader() string {
+	var output string
+
+	if cmd.FullSemanticMode {
+		// Add import
+		output += `
+import "owl/owl.proto";
 
+`
+		// Prepare prefix output
+		var prefixOutputs []string
+
+		for short, prefix := range cmd.preparedOntology.Prefixes {
+			prefixOutputs = append(prefixOutputs, `{
+	prefix: "`+short+`"
+	iri: "`+prefix.IRI+`"
+}`)
+		}
+
+		// Add ontology meta-data
+		output += `
+option (owl.meta) = {
+	prefixes: [` + strings.Join(prefixOutputs, ",") + `]};
+`
+
+	} else {
+		// Add EnumValueOptions
+		output += `
+extend google.protobuf.MessageOptions {
+	repeated string resource_type_names = 60000;
+}`
+
+	}
+
+	return output
+}
+
+func (cmd *GenerateProtoCmd) emitClassOptions(iri string) string {
```

Comment:
```suggestion
// emitClassOptions adds the class options IRI and parent when full semantic mode is enabled, otherwise it adds only the resource type name.
func (cmd *GenerateProtoCmd) emitClassOptions(iri string) string {
```